src/zodbupdate/serialize.py

```diff
@@ -298,18 +293,17 @@ def rename(self, input_file):
         return None otherwise.
         """
         self.__changed = False
-        self.__skipped = False
 
         unpickler = self.__unpickler(input_file)
         class_meta = unpickler.load()
```

Comment:
We should the if self.__skipped after this line, and the old implementation will work as expected, and add a test for the reference.